app/lib/dags/experimental.json.erb

```diff
@@ -7,6 +7,18 @@
       "gsnap.hitsummary.tab",
       "rapsearch2.hitsummary.tab"
     ],
+    <% if attribute_dict[:pipeline_version].to_f >= 3.6 %>
```

Comment:
this would fail if we push a "3.10"?

Reply:
This check will get much more verbose when we get to 3.10...